docs/xproc/xproc.md

```diff
@@ -1,149 +1,149 @@
+For instance, an example of a simple XProc processing pipeline is processing an XML document by first resolving the XInclude calls, transform it with several consecutive XSL stylesheets, and then validate it against an XML Schema. When it is valid, it is stored to disk. More complex XProc pipelines could involve splitting and merging documents, working with zip files (through extensions), making processing decisions based on XPath (2.0) expressions, etc.
 
-There are several implementations of XProc, but the most used is probably Calabash. eXist (V2.1) currently supports XProc using Calabash through its xmlcalabash extension module. This is an experimental implementation and does not by far unleash all XProc’s protential. The same is true for a module called xproc that tries to implement XProc using XQuery.
+There are several implementations of XProc, but the most widely used is probably Calabash. eXist currently supports XProc using Calabash through its xmlcalabash extension module. This is an experimental implementation and does not by far unleash all XProc’s protential. The same is true for a module called xproc that tries to implement XProc using XQuery.
 
 To amend this and add XProc to the suite of X technologies supported by eXist, a new XProc module was developed in 2013/2014.
 
-## Installing and de-installing
-The XProc module will only work on eXist versions > V2.1! That is, as long as the official download version is still V2.1, you need the most current version from eXist’s code repos itory. Refer to [http://exist-db.org/exist/apps/doc/building.xml](http://exist-db.org/exist/apps/doc/building.xml) for more information.
+## Installation and removal
+
+The XProc module requires eXist 3.0.
 
+### Installation
 
-### Install
+Install it via eXist's Dashboard > Package Manager. 
 
-Fire up the package manager from eXist’s dashboard and install it from the public eXist module repository. 
+### Removal
 
-### Deinstall
-De-installing the package is not the easy normal package manager’s routine for this. Because the package includes a jar file, it is (partly) locked by eXist and some extra work is required:
+Uninstalling the package is not the easy normal package manager’s routine for this. Because the package includes a jar file, it is (partly) locked by eXist and some extra work is required:
 
-1. De-install the package first through the package manager’s normal de-install command. This seems to work, but when you refresh the list the package still seems to be there.
+1. Uninstall the package first through the Package Manager. This seems to work, but when you refresh the list the package still seems to be there.
 1. Stop eXist 
```

Comment:
package manager should do that, yes; But I have been repairing this recently manually. Though it might be a version of 'develop' from mid-may last year, so things are fixed now?